### DISCUSSION TITLE
نظرتون چیه یه پروژه با هم بریم ؟

### DISCUSSION THREAD

**Discussion by good-boy01** [Helli1D38/About-us#2] [Polls]
یه پروژه با هم بریم مثلا ساخت وب سایت برا مدرسه یا انجمنا یا هر چیز دیگه ای

**Comment by seniyorsina** [comment]
> یه پروژه با هم بریم مثلا ساخت سایت برا مدرسه یا انجمنا یا هر چیز دیگه ای
شما انجام بدید منم نگاه میکنم:کاکا سنگی:

**Comment by good-boy01** [comment]
> > یه پروژه با هم بریم مثلا ساخت سایت برا مدرسه یا انجمنا یا هر چیز دیگه ای
> شما انجام بدید منم نگاه میکنم:کاکا سنگی:
> 

**باش🗿**

**Comment by BGOPC** [comment]
بریم
